static/bidder-info/pubmatic.yaml

```diff
@@ -11,7 +11,11 @@ capabilities:
       - banner
       - video
 userSync:
+  default: "iframe"
   iframe:
     url: "https://ads.pubmatic.com/AdServer/js/user_sync.html?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&predirect={{.RedirectURL}}"
     userMacro: ""
     # pubmatic appends the user id to end of the redirect url and does not utilize a macro
+  redirect:
+    url: "https://image8.pubmatic.com/AdServer/ImgSync?p=159706&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&pu={{.RedirectURL}}"
+    userMacro: "#PMUID"
```

Comment:
I tested this redirect url, but do not see the `pu` redirect url honored. Please verify this functionality.

Reply:
I tested this again today and it looks to be working fine. Not sure what was going on yesterday with our tests.

Reply:
Yes, I see correct behavior today again.